src/components/SidebarContent.js

```diff
@@ -1,32 +1,38 @@
 import React from "react";
-import { storeCustomers } from "../data";
+// import { storeCustomers } from "../data";
 import { CustomerConsumer } from "./CustomerContext";
 
 export default function SidebarContent() {
   return (
     <div className="side-menu-container">
       <CustomerConsumer>
         {(ctx) => {
+          let customer = ctx.customers.filter(
+            (customer) => customer.name === ctx.customer
+          )[0];
```

Comment:
see? you keep copy-pasting it all over the place

Reply:
the easiest (not the best from a tech point of view, though) would be to add a function: `ctx.getCustomer()` which does this: `this.state.customers.find((customer) => this.state.customer === customer.name)`